Move settings to table

Devin-Holland opened the pull request.

kylebernhardy commented:
To write settings are we relying on the native REST interface to the PrometheusExporterSettings table?  There is still a function that is pulling the settings file `getSettings` I'm assuming this is now an orphan?

Devin-Holland commented:
Yes we would use the native REST endpoints for `prometheus_exporter/PrometheusExporterSettings` Anyone with write permissions to the table would be allowed to modify the settings. By default it should only be the HDB_ADMIN user.

I will get the `getSettings()` method removed and update the PR when it's done.

kylebernhardy commented:
With that in mind the `@export` declaration will need to be added to the table in the `schema.graphql`.  Also documentation should be added defining the setting we look for and sample values so current/future users will understand usage.

Devin-Holland commented:
> With that in mind the `@export` declaration will need to be added to the table in the `schema.graphql`. Also documentation should be added defining the settings we look for and sample values so current/future users will understand usage.

Since it's exported here: https://github.com/HarperDB-Add-Ons/prometheus_exporter/pull/9/files#diff-77dd88b955c88b167f40f2e14c95f0be029315ae537390cd1deced862cca033dL321 

It doesn't need to be exported. Unless we don't want it at `prometheus_exporter/PrometheusExporterSettings` and instead just want `PrometheusExporterSettings`. I'm open to either.

But I can document it in the README.

Devin-Holland commented:
I added some examples and descriptions to the README and cleaned up some more orphaned code from the old settings method. 

I left the export at `/prometheus_exporter/PrometheusExporterSettings`. Let me know if you want me to move that.

kylebernhardy commented:
This is great! Thanks for working on this.
